[2022-07-29 10:54:04,014] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Second_one.first_task scheduled__2021-10-26T00:00:00+00:00 [queued]>
[2022-07-29 10:54:04,029] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Second_one.first_task scheduled__2021-10-26T00:00:00+00:00 [queued]>
[2022-07-29 10:54:04,029] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 10:54:04,029] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-07-29 10:54:04,029] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 10:54:04,052] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): first_task> on 2021-10-26 00:00:00+00:00
[2022-07-29 10:54:04,057] {standard_task_runner.py:52} INFO - Started process 2599 to run task
[2022-07-29 10:54:04,060] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Second_one', 'first_task', 'scheduled__2021-10-26T00:00:00+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/Second_One.py', '--cfg-path', '/tmp/tmpjhd2_7p9', '--error-file', '/tmp/tmpw6xjhqru']
[2022-07-29 10:54:04,060] {standard_task_runner.py:80} INFO - Job 92: Subtask first_task
[2022-07-29 10:54:04,128] {task_command.py:369} INFO - Running <TaskInstance: Second_one.first_task scheduled__2021-10-26T00:00:00+00:00 [running]> on host 9f0f006fc89d
[2022-07-29 10:54:04,229] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yosr
AIRFLOW_CTX_DAG_ID=Second_one
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-26T00:00:00+00:00
[2022-07-29 10:54:04,230] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-29 10:54:04,231] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hello world']
[2022-07-29 10:54:04,245] {subprocess.py:85} INFO - Output:
[2022-07-29 10:54:04,248] {subprocess.py:92} INFO - hello world
[2022-07-29 10:54:04,248] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-29 10:54:04,292] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Second_one, task_id=first_task, execution_date=20211026T000000, start_date=20220729T105404, end_date=20220729T105404
[2022-07-29 10:54:04,321] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-29 10:54:04,394] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
